tests/unittests/text/test_sacre_bleu.py

```diff
@@ -28,9 +28,6 @@
     from sacrebleu.metrics import BLEU
 
 
-TOKENIZERS = ("none", "13a", "zh", "intl", "char")
-
-
 def _sacrebleu_fn(preds: Sequence[str], targets: Sequence[Sequence[str]], tokenize: str, lowercase: bool) -> Tensor:
     sacrebleu_fn = BLEU(tokenize=tokenize, lowercase=lowercase)
```

Comment:
looks like tests are failing because the reference does not support all the new tokenizers

Reply:
Yes, some older versions of `SacreBLEU` do not have support for new tokenizers. Will fix this week :]